website/source/guides/advanced-scheduling/spread.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Spread"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+    The spread stanza allows operators to increase failure tolerance in their Nomad clusters by allowing them to distribute their workloads in a customized way based on attributes or client metadata.
+---
+
+# Increasing Failure Tolerance with Spread
+
+The Nomad scheduler uses a bin packing algorithm when making job placements on
+nodes to optimize resource utilization and density of applications. Although
+this feature ensures that cluster resources are being used efficiently, it does
+not necessarily promote maximum failure tolerance of jobs across nodes.
+
+The `spread` stanza solves this problem by allowing operators to distribute their workloads in a customized way based on [attributes][attributes] and/or [client metadata][client-metadata]. By implementing spread, Nomad operators can ensure maximum levels of failure tolerance based on their specific architectures.
```

Comment:
Suggested replacement for "By implementing spread".. "By using spread criteria in their job specification, Nomad job operators can ensure that failures across a domain such as datacenter or rack don't affect application availability"